server/project/TypescriptBuilder.scala

```diff
@@ -54,18 +56,18 @@ object TypescriptBuilder extends AutoPlugin {
     compileTypescript := { inputFiles: Seq[PathMapping] =>
       val streamsVal = (Assets / streams).value: @sbtUnchecked
       // targetDir will contain compiled JS files.
-      val targetDir = new File(webTarget.value, "typescript/javascripts")
+      val targetDir = new File(webTarget.value, "dist")
       val cacheDir = new File(streamsVal.cacheDirectory, "run")
       val compiledJsFiles: Seq[File] = recompileTypescriptIfFilesChanged(
         inputFiles,
         targetDir,
         cacheDir,
         streamsVal.log
       )
-      // transform compiles JS files to PathMapping and add "javascripts" as part of the file name.
-      // That way JS files fille be added in /assets/javascripts/*.js in final output.
+      // transform compiles JS files to PathMapping and add "dist" as part of the file name.
+      // That way JS files fille be added in /assets/dist/*.js in final output.
       val jsFiles: Seq[PathMapping] =
-        compiledJsFiles.map({ f => (f, "javascripts/" + f.getName) })
+        compiledJsFiles.map({ f => (f, "dist/" + f.getName) })
```

Comment:
question (blocking): why rename the output from `javascripts` to `dist`?

Reply:
I renamed it because the output is no longer only JS.  The compiled CSS file is output there too.  `dist` seemed like an appropriate name given that it's the default Webpack output file name.  It also seemed like a clearer name for bundling output because it differentiates it from source code.

I attempted to find a way to split the output into two different directories, but wasn't able to find a way to configure that in Webpack.  It also seemed unnecessary.

Reply:
Thanks, that makes sense to me! Sounds like an apt rename. I think I disagree with the decision to leave this as `TypescriptBuilder` given it's now responsible for stylesheets as well.